src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php

```diff
@@ -105,6 +144,27 @@ public function GetFrontendActionUrlToSendOrderEmail($sMail = null)
         return $sURL;
     }
 
+    public function exportOrderToWaWi(): array
```

Comment:
The name should not include the german "WaWi" abbreviation even if the methods it is calling was misnamed.
We should call it exportOrders() instead. We don't know if it is exported to an inventory system or something else (could be an FTP transfer, email etc.). The ExportOrderForWaWiHook is misnamed, but we can't rename it without BC breaks.

Reply:
Renamed